portals/management-portal/README.md

```
@@ -1,73 +1,195 @@
+
+Once these steps are complete, the Management Portal will be running.
+
+---
+
+# 5️⃣ Important Pre‑requisites
+
+### ✅ Allow Backend HTTPS Certificate in Browser
+
+Visit:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)